sites/example-project/src/components/viz/ECharts.svelte

```diff
@@ -34,7 +36,8 @@
 
 <EchartsCopyTarget {config} {height} {width} {copying}/> 
 
-<DownloadData class=download-button downloadData={() => {downloadChart = true; setTimeout(() => { downloadChart = false}, 0);}} />
+<DownloadData text="Download data" {data} class=download-button />
+<DownloadData text="Save Chart" class=download-button downloadData={() => {downloadChart = true; setTimeout(() => { downloadChart = false}, 0);}} />
```

Comment:
Let's be consistant here with capitalization vs "Download data". 

Also suggest we say "Save image", which I think makes it 100% explicit what the format is. 

I wonder also if there is a better icon we can find here. Perhaps something like this: ![image](https://user-images.githubusercontent.com/58074498/207755335-d3263c92-f03c-4b34-8c5c-9e015c4a9676.jpeg) 

There are probably some options in the same icon library as the download icon

Reply:
Changed to 'Save image' and applied the image icon.

https://user-images.githubusercontent.com/14843458/207907624-29a166bc-79b4-4ebf-93c7-6536a7da83e4.mov